app/models/spree/order_decorator.rb

```diff
@@ -0,0 +1,13 @@
+module Spree
+  module OrderDecorator
+    def self.prepended(base)
+      base.has_many :commissions, class_name: 'Spree::OrderCommission'
+    end
+
+    def finalize!
```

Comment:
@pmilewski rather than overriding finalize method, let's use checkout state machine:

https://guides.spreecommerce.org/developer/customization/checkout.html#modifying-the-checkout-flow